From e63ffe0c00c45e381f92e1eaffae07cd15db1f83 Mon Sep 17 00:00:00 2001
From: "tao.dong" <tao.dong@amlogic.com>
Date: Mon, 3 Nov 2014 14:55:25 +0800
Subject: [PATCH 5011/5965] PD#99263:Impove temperature threshold to avoid
 reboot problem under high temperature

If temp sensor report temperature over 110 degree, system will reboot,So increase threshold of reboot
 protect temprature to max measure range of temperature sensor to avoid this problem.

Change-Id: I8a83cc9db9c8990e98fb1cf0f4bd8dcfffbee7d7
---
 arch/arm/boot/dts/amlogic/meson8_k100_1G.dtd                   | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k100_2G.dtd                   | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k101_1G.dtd                   | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k101_512M.dtd                 | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k102_v1.dtd                   | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200a_1G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200a_1G_hdmiin.dtd           | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200a_2G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200a_2G_4k2k.dtd             | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200a_2G_hdmiin.dtd           | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc.dtd        | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc_hdmiin.dtd | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio.dtd        | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio_hdmiin.dtd | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_2G_4k2k.dtd             | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdhc.dtd        | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdio.dtd        | 2 +-
 arch/arm/boot/dts/amlogic/meson8_k200b_2G_hdmiin.dtd           | 2 +-
 arch/arm/boot/dts/amlogic/meson8_pxp.dtd                       | 2 +-
 arch/arm/boot/dts/amlogic/meson8_skt.dtd                       | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m100_1G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd                | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m200_1G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m200_1G_iptv.dtd             | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m201C_512M.dtd               | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m201_1G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m201_512M.dtd                | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m202_512M.dtd                | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m203a.dtd                    | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_m203b.dtd                    | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_pxp.dtd                      | 2 +-
 arch/arm/boot/dts/amlogic/meson8b_skt.dtd                      | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_n01_2G.dtd                  | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_n100_1G.dtd                 | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_n101_512M.dtd               | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_n200_1G.dtd                 | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_n200_2G.dtd                 | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_pxp.dtd                     | 2 +-
 arch/arm/boot/dts/amlogic/meson8m2_skt.dtd                     | 2 +-
 40 files changed, 40 insertions(+), 40 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/meson8_k100_1G.dtd b/arch/arm/boot/dts/amlogic/meson8_k100_1G.dtd
index 8d958f7494cd..fcff2d31a3dc 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k100_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k100_1G.dtd
@@ -1908,7 +1908,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k100_2G.dtd b/arch/arm/boot/dts/amlogic/meson8_k100_2G.dtd
index bc0bc7405602..da19ba03b29a 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k100_2G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k100_2G.dtd
@@ -1918,7 +1918,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k101_1G.dtd b/arch/arm/boot/dts/amlogic/meson8_k101_1G.dtd
index dad766efc101..36aea4b6bc9a 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k101_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k101_1G.dtd
@@ -1881,7 +1881,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k101_512M.dtd b/arch/arm/boot/dts/amlogic/meson8_k101_512M.dtd
index e0ff70994973..0ed1d09368eb 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k101_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k101_512M.dtd
@@ -1809,7 +1809,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k102_v1.dtd b/arch/arm/boot/dts/amlogic/meson8_k102_v1.dtd
index fa2028858a3d..62f462ca36a7 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k102_v1.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k102_v1.dtd
@@ -1832,7 +1832,7 @@ lcd_LD070WX4:lcd_LD070WX4{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200a_1G.dtd b/arch/arm/boot/dts/amlogic/meson8_k200a_1G.dtd
index 27f745060b4c..53b49f9df1c2 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200a_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200a_1G.dtd
@@ -1029,7 +1029,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200a_1G_hdmiin.dtd b/arch/arm/boot/dts/amlogic/meson8_k200a_1G_hdmiin.dtd
index 5968bcf1226d..37ddd1f4501e 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200a_1G_hdmiin.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200a_1G_hdmiin.dtd
@@ -1025,7 +1025,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200a_2G.dtd b/arch/arm/boot/dts/amlogic/meson8_k200a_2G.dtd
index 3681f0058861..6ceb53a706ec 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200a_2G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200a_2G.dtd
@@ -913,7 +913,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200a_2G_4k2k.dtd b/arch/arm/boot/dts/amlogic/meson8_k200a_2G_4k2k.dtd
index 08d70f0c4654..b5ae9d62ec48 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200a_2G_4k2k.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200a_2G_4k2k.dtd
@@ -890,7 +890,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200a_2G_hdmiin.dtd b/arch/arm/boot/dts/amlogic/meson8_k200a_2G_hdmiin.dtd
index 177d21d2acd6..a54fd7efb1ef 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200a_2G_hdmiin.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200a_2G_hdmiin.dtd
@@ -895,7 +895,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc.dtd
index 9e6fc7dccf6e..845c1d3b01e4 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc.dtd
@@ -1043,7 +1043,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc_hdmiin.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc_hdmiin.dtd
index 8f3341ecabac..0ce952ce5593 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc_hdmiin.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdhc_hdmiin.dtd
@@ -1041,7 +1041,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio.dtd
index 48fade445067..a85d958873c5 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio.dtd
@@ -1030,7 +1030,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio_hdmiin.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio_hdmiin.dtd
index c0170d04462d..02d74f63959d 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio_hdmiin.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_1G_emmc_sdio_hdmiin.dtd
@@ -1029,7 +1029,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_4k2k.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_4k2k.dtd
index 276782388b11..b4ad0e69038f 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_4k2k.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_4k2k.dtd
@@ -983,7 +983,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdhc.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdhc.dtd
index ba7f305e83fa..cab692d8f9ea 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdhc.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdhc.dtd
@@ -1008,7 +1008,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdio.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdio.dtd
index 1affce2e20c4..c760c6926f46 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdio.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_emmc_sdio.dtd
@@ -1006,7 +1006,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_hdmiin.dtd b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_hdmiin.dtd
index af0b91fce140..9722050d9461 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k200b_2G_hdmiin.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k200b_2G_hdmiin.dtd
@@ -907,7 +907,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_pxp.dtd b/arch/arm/boot/dts/amlogic/meson8_pxp.dtd
index ba6bc3153ea7..6c15a87cb213 100755
--- a/arch/arm/boot/dts/amlogic/meson8_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_pxp.dtd
@@ -1083,7 +1083,7 @@ void root_func(){
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8_skt.dtd b/arch/arm/boot/dts/amlogic/meson8_skt.dtd
index 6cb468037fbb..2cf909abb646 100755
--- a/arch/arm/boot/dts/amlogic/meson8_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_skt.dtd
@@ -1214,7 +1214,7 @@ void root_func(){
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m100_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m100_1G.dtd
index 0cdbaa3576b8..07a8f40cd916 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m100_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m100_1G.dtd
@@ -1701,7 +1701,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
index b19d2d6825ad..1b367ecd11f2 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
@@ -1575,7 +1575,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
index 9267c8dbd788..a559f5260f41 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
@@ -1681,7 +1681,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m200_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m200_1G.dtd
index 85749f463baf..be4b1688a27c 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m200_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m200_1G.dtd
@@ -1258,7 +1258,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m200_1G_iptv.dtd b/arch/arm/boot/dts/amlogic/meson8b_m200_1G_iptv.dtd
index d7df5d0c7c4d..700aec8bb539 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m200_1G_iptv.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m200_1G_iptv.dtd
@@ -1258,7 +1258,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m201C_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m201C_512M.dtd
index 93d92a61ae94..9b3da16551da 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m201C_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m201C_512M.dtd
@@ -1305,7 +1305,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m201_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m201_1G.dtd
index 4987d169413c..384f8a2ec17c 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m201_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m201_1G.dtd
@@ -1314,7 +1314,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m201_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m201_512M.dtd
index 9077f890047d..8874f82abf34 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m201_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m201_512M.dtd
@@ -1316,7 +1316,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m202_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m202_512M.dtd
index ff23d55c7d9f..fcd70324d579 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m202_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m202_512M.dtd
@@ -1200,7 +1200,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m203a.dtd b/arch/arm/boot/dts/amlogic/meson8b_m203a.dtd
index 1dac214dfde1..bc05efa532d2 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m203a.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m203a.dtd
@@ -1218,7 +1218,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m203b.dtd b/arch/arm/boot/dts/amlogic/meson8b_m203b.dtd
index f364c36679ec..1cacd5cce3f6 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m203b.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m203b.dtd
@@ -1218,7 +1218,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_pxp.dtd b/arch/arm/boot/dts/amlogic/meson8b_pxp.dtd
index 2ca33ac73b5b..bffd002068c0 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_pxp.dtd
@@ -1226,7 +1226,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8b_skt.dtd b/arch/arm/boot/dts/amlogic/meson8b_skt.dtd
index 06e4b3994e18..7c99aba2697b 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_skt.dtd
@@ -1325,7 +1325,7 @@ void root_func(){
         trip_point=<70 1488001 1488001 511 511 3 2
                     80 1200001 1200001 435 435 2 2
                     90  800001  800001 328 328 1 1
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_n01_2G.dtd b/arch/arm/boot/dts/amlogic/meson8m2_n01_2G.dtd
index 185b9e282681..3957f8acfd44 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_n01_2G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_n01_2G.dtd
@@ -544,7 +544,7 @@
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
             cpu_cali_a=<0>;
             idle_interval=<1000>;
     use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_n100_1G.dtd b/arch/arm/boot/dts/amlogic/meson8m2_n100_1G.dtd
index cd4f7cdf734f..5cc251cd733f 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_n100_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_n100_1G.dtd
@@ -1901,7 +1901,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_n101_512M.dtd b/arch/arm/boot/dts/amlogic/meson8m2_n101_512M.dtd
index b8d970f540ea..db163ba4a99f 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_n101_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_n101_512M.dtd
@@ -1808,7 +1808,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_n200_1G.dtd b/arch/arm/boot/dts/amlogic/meson8m2_n200_1G.dtd
index 77e54f8e09ca..a2b1929bfa40 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_n200_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_n200_1G.dtd
@@ -1008,7 +1008,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_n200_2G.dtd b/arch/arm/boot/dts/amlogic/meson8m2_n200_2G.dtd
index 765f427d6c14..27a462c39e20 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_n200_2G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_n200_2G.dtd
@@ -1008,7 +1008,7 @@ sdio{
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_pxp.dtd b/arch/arm/boot/dts/amlogic/meson8m2_pxp.dtd
index 524c5b1d2c91..a0467c081509 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_pxp.dtd
@@ -1238,7 +1238,7 @@ void root_func(){
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
diff --git a/arch/arm/boot/dts/amlogic/meson8m2_skt.dtd b/arch/arm/boot/dts/amlogic/meson8m2_skt.dtd
index f35c5dac78d2..af89ed9a9be5 100755
--- a/arch/arm/boot/dts/amlogic/meson8m2_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8m2_skt.dtd
@@ -1214,7 +1214,7 @@ void root_func(){
         trip_point=<70 1608001 1608001 511 511 3 5 
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
-                    110 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
+                    260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
 		cpu_cali_a=<0>;
 		idle_interval=<1000>;
         use_virtual_thermal;
-- 
2.19.0

